db/migrate/20150413043822_add_settings_to_instances_and_courses.rb

```diff
@@ -0,0 +1,6 @@
+class AddSettingsToInstancesAndCourses < ActiveRecord::Migration
+  def change
+    add_column :instances, :settings, :text
```

Comment:
Actually can, cause if you don't have any settings, it's a empty hash <code> {} </code> by default. 

But I think the gem should handle this column even it's <code>null</code> ? We should not know the abstractions in gem.